labelImg.py

```diff
@@ -849,21 +878,18 @@ def format_shape(s):
             if self.label_file_format == LabelFileFormat.PASCAL_VOC:
                 if annotation_file_path[-4:].lower() != ".xml":
                     annotation_file_path += XML_EXT
-                self.label_file.save_pascal_voc_format(annotation_file_path, shapes, self.file_path, self.image_data,
-                                                       self.line_color.getRgb(), self.fill_color.getRgb())
+                self.label_file.save_pascal_voc_format(annotation_file_path, shapes, self.file_path, self.image_data)
```

Comment:
Rationale: The line and fill colors are not supported by the voc format, so I removed the call here.

This may be relevant for future changes to the code, but I followed the YAGNI principle here